docs/ReleaseNewVersion.md

```diff
@@ -53,6 +53,6 @@ to be created manually. First build PHAR file manually:
 $ make build
 ```
 
-Once the above command suceeded, navigate to latest release in [GitHub
+Once the above command succeeded, navigate to latest release in [GitHub
```

Comment:
`succeeds` or `has succeeded` may be more appropriate.

Reply:
Thanks, @junpataleta . Amended.